lib/WebService/SendBird.pm

```diff
@@ -0,0 +1,306 @@
+package WebService::SendBird;
+
+use strict;
+use warnings;
+
+use Mojo::UserAgent;
+use Mojo::URL;
+
+use WebService::SendBird::User;
+use WebService::SendBird::GroupChat;
+
+use Carp qw();
+
+our $VERSION = '0.001';
+
+# ABSTRACT: client to work with SendBird API
+
+=head1 NAME
+
+WebService::SendBird - unofficial support for the Sendbird API
+
+=head1 SYNOPSIS
+
+ use WebService::SendBird;
+
+ my $api = WebService::SendBird->new(
+     api_token => 'You_Api_Token_Here',
+     app_id    => 'You_App_ID_Here',
+ );
+
+ my $user = $api->create_user(
+     user_id     => 'my_chat_user_1',
+     nickname    => 'pumpkin',
+     profile_url => undef,
+ );
+
+ my $chat = $api->create_group_chat(
+     user_ids => [ $user->user_id ],
+ );
+
+=head1 DESCRIPTION
+
+Basic implementation for SendBird Platform API client, which helps to create users and group chats.
+
+More information at L<Platform API Documentation|https://docs.sendbird.com/platform>
+
+=cut
+
+use constant DEFAULT_API_URL_TEMPLATE => 'https://api-%s.sendbird.com/v3';
+use constant DEFAULT_REQUEST_TIMEOUT  => 15;
+
+=head1 METHODS
+
+=head2 new
+
+Creates an instance of API client
+
+=over 4
+
+=item * C<api_token> - Master or Secondary API Token.
+
+=item * C<app_id> - Application ID.
```

Comment:
It could be confused with our Binary application id.
```suggestion
=item * C<app_id> - Sendbird Application ID ot be used for constructing the base URL.
```